we need a fixation for sqlalchemy

because 2.0 was released and we are not compatible yet

```
==================================== ERRORS ====================================
__________ ERROR collecting tests/_test_mscolab/test_chat_manager.py ___________
tests/_test_mscolab/test_chat_manager.py:31: in <module>
    from mslib.mscolab.models import Message, MessageType
mslib/mscolab/models.py:37: in <module>
    db = SQLAlchemy()
/opt/conda/envs/mssenv/lib/python3.10/site-packages/flask_sqlalchemy/__init__.py:758: in __init__
    _include_sqlalchemy(self, query_class)
/opt/conda/envs/mssenv/lib/python3.10/site-packages/flask_sqlalchemy/__init__.py:112: in _include_sqlalchemy
    for key in module.__all__:
E   AttributeError: module 'sqlalchemy' has no attribute '__all__'. Did you mean: '__file__'?
__________ ERROR collecting tests/_test_mscolab/test_file_manager.py ___________
```

--- a/localbuild/meta.yaml
+++ b/localbuild/meta.yaml
@@ -23,6 +23,11 @@ requirements:
   build:
     - python
     - pip
+    - setuptools ==65.5.1
+  host:
+    - python
+    - setuptools ==65.5.1
+    - pip
     - future
     - menuinst  # [win]
   run:

--- a/localbuild/meta.yaml
+++ b/localbuild/meta.yaml
@@ -59,6 +59,7 @@ requirements:
     - flask-httpauth
     - flask-mail
     - flask-migrate
+    - sqlalchemy <2.0.0
     - werkzeug >2.0.0
     - flask-socketio =5.1.0
     - flask-sqlalchemy

